lib/optimizely/decision_service.rb

```diff
@@ -242,7 +240,7 @@ def get_variation_for_feature_rollout(feature_flag, user_id, attributes = nil)
         )
         # Evaluate if user satisfies the traffic allocation for this rollout rule
         variation = @bucketer.bucket(rollout_rule, bucketing_id, user_id)
-        return variation unless variation.nil?
+        return Decision.new(rollout_rule, variation, DECISION_SOURCE_ROLLOUT) unless variation.nil?
```

Comment:
Please update method def return for this method. The method does not return variation any more